dhall-nixpkgs/dhall-nixpkgs.cabal

```diff
@@ -19,7 +19,7 @@ Executable dhall-to-nixpkgs
   Build-Depends:       base                 >= 4.11     && < 5
```

Comment:
Why do we don't do a patch upgrade of this package version?

Reply:
Oh, is it the first release?

Reply:
> Why do we don't do a patch upgrade of this package version?

I mean the `dhall-nixpkgs` package version

Reply:
Yeah, `dhall-nixpkgs` hasn't been released yet.  This will be the first release.